docs/pages/reference/architecture/teleport-cloud-architecture.mdx

```diff
@@ -47,8 +47,10 @@ own AWS account where retention period can be managed independently.
 ## High Availability
 
 ### Auth Service
-The Teleport [Auth Service](authentication.mdx) is deployed in 2 AWS availability zones. Single or multi-region zone
-deployments are available as options for fault tolerance. AWS guarantees [99.99%](https://aws.amazon.com/compute/sla/) of monthly uptime.
+The Teleport [Auth Service](authentication.mdx) is deployed to two AWS availability zones in a single region.
```

Comment:
I wouldn't link to `authentication.mdx` here, since that document is about authentication in general, not about the Teleport Auth Service.

Reply:
thanks @zmb3 , I've updated that to the architecture section that introduces the auth service.